### PR TITLE
Rank 2 for all nations no longer capped

### PR DESCRIPTION
**Pull request by kaincenteno**
Mission 2-3 BCNM
The Three Kingdoms, Journey Abroad, The Emissary

### CODE REVIEW

File: sql/bcnm_info.sql [hunk 1]
```diff
@@ -45,7 +45,7 @@ CREATE TABLE `bcnm_info` (
 LOCK TABLES `bcnm_info` WRITE;
 /*!40000 ALTER TABLE `bcnm_info` DISABLE KEYS */;
 set autocommit=0;
-INSERT INTO `bcnm_info` VALUES (0,139,'rank_2_mission','Shitstain',1,18,1800,25,6,0,5);
+INSERT INTO `bcnm_info` VALUES (0,139,'rank_2_mission','Shitstain',1,18,1800,99,6,0,5);
```
Comment (davismj) [inline sql/bcnm_info.sql]: This is technically a level cap of 99, no uncapped.

**Comment by helixhamin** [comment]
@davismj Agreed. If there was a way to toggle the change, then it would be different. 

Makes me wonder, we have:
`#Enable/disable level cap of Chains of Promathia mission battlefields stored in database.
CoP_Battle_cap: 1` in the map conf file, why not the nations battlefields?

**Comment by TeoTwawki** [comment]
we did a setting for this for the CoP battlefields. At the time nation 2-3 was still capped. maybe rename the setting and combine them? old_school_mission_caps or something.

Actually, the way I did the setting was crappy. :thinking: I have an idea I will look into after work today.

**Comment by TeoTwawki** [comment]
ok so, I was thinking of adding a column for what expansion a battlefield comes from, and then instead of checking the BCNM ID as the existing CoP_Battle_cap does, we could per-expansion enable/disable the caps.

**Comment by TeoTwawki** [comment]
er, I no forget that. A column for "is this a mission", and then enable/disable them via checking that instead of ID

Id rather not have a half dozen settings when 1 will do.

**Comment by TeoTwawki** [comment]
#4988

**Comment by takhlaq** [comment]
https://github.com/DarkstarProject/darkstar/pull/4988

**Pull request closed without merging**